modules/services/buildkite-agents.nix

```
@@ -228,7 +228,8 @@ in
       { path = cfg.runtimePackages ++ [ cfg.package pkgs.coreutils pkgs.darwin.DarwinTools ];
         environment = {
           HOME = cfg.dataDir;
-        }// (if config.nix.useDaemon then { NIX_REMOTE = "daemon"; } else {});
+          NIX_SSL_CERT_FILE = "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
```

I think it would be preferable to inherit it

```suggestion
          inherit (config.environment.variables) NIX_SSL_CERT_FILE;
```

---

That's indeed a nice way to avoid the duplication, looks good otherwise. Launchd as an isolated environment so any service that needs tls connections will need this in it's environment.

---

Somehow missed this thread, updated my PR